### ISSUE TITLE
Rating stars to use the numbers?

### ISSUE THREAD

**Issue by socialblogsite** [FortAwesome/Font-Awesome#6679]
Considering FontAwesome is not using the numbers, could it include icons that actually NEED to be read as numbers? (e.g. by screen readers). Like Rating Stars! :-)

I invented this system with fonts years ago, to render stars from actual decimal numbers.
http://socialblogsitewebdesign.com/responsive-rating-stars/

You can even use 2.4/3 (which means 2.4 out of 3) to change the base outline to three stars, rather than the usual 5, or using different colors for the integer…

It doesn't use extra divs, images scripts or canvas. Only spans if you decide to add a forward slash with the base stars other than 5 (the default).

**Comment by tagliala** [comment]
Good Idea. Since we will probably need an `fa-sr-only` class (for screen readers only text), we could improve that example. Refers to #6133

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
